test/tests/userobjects/layered_side_integral_functor/layered_side_integral_functor.i

```diff
@@ -0,0 +1,74 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 40
+  ny = 10
+  nz = 10
+  allow_renumbering = false
+[]
+
+[Materials]
+  [u_mat]
+    type = ADGenericFunctorMaterial
```

Comment:
```suggestion
    type = GenericFunctorMaterial
```
Both this and the similar suggested changes are optional. Strictly speaking since the function is a non-AD functor, there is no need for the mat prop to be an AD functor